### DISCUSSION TITLE
Unable to connect to the database: ConnectionError [SequelizeConnectionError]: connect ETIMEDOUT     at ConnectionManager.connect (/Users/macbookpro/Documents/projects/envoy-api/node_modules/sequelize/lib/dialects/mysql/connection-manager.js:102:17)

### DISCUSSION THREAD

**Discussion by sajidbrong** [community/community#64702] [Programming Help] [answered]
### Body

I am getting error while running bull js cron job is there any redis error related to this.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by arafajs** [comment]
- Xamp/ Mamp must be online. Default ports will be taken by this local server, so try a different port for mysql 8889 instead of 3306.
- Ensure mysql server is running in the host you are trying to connect to.
- Ensure the host ip is correct.
- Ensure that the port entered is correct.
- Ensure that the firewall rules are defined correctly.

**Comment by bronglil** [comment, marked as answer]
The error message you're encountering indicates that your Sequelize-based application is having trouble connecting to the MySQL database. The "ETIMEDOUT" error typically indicates that the connection attempt to the database server timed out. Here are steps you can take to resolve this issue


**Check MySQL Server Status:**
Ensure that your MySQL server is up and running. Verify that it's accepting connections and not facing any issues.

**Network Connectivity:**
Make sure your application server can reach the MySQL server. Check for any network issues or firewalls that might be blocking the connection.

**Check Database Configuration:**
Verify that the database configuration in your Sequelize application is correct. Double-check the host, port, username, password, and database name settings.


This is not related to the redis issue this is related to the DB connection with you local machine.

**Reply by Yusein96** [reply]
780722****
